### DISCUSSION TITLE
article/richang/20230917-vercel-china-dns

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#6] [Announcements]
# article/richang/20230917-vercel-china-dns

一招解决Vercel站点在国内无法访问的问题 最近一直在公众号等渠道推我们的Chrome插件：Youtube中文配音，经常收到类似无法访问，网站挂掉的反馈： 其实，网站一直没关，只是一个静态站点，没那么容易挂的。只是Vercel部署的站点都被DNS污染了，导致网站在中国大陆是无法访问的。用户端处理的话，手工设置下DNS为8.8.8.8就可以访问。

https://www.didispace.com/article/richang/20230917-vercel-china-dns.html

**Comment by leoshcn** [comment]
请问这个查询 域名 dns 污染的服务是什么？

**Reply by dyc87112** [reply]
这个：https://tool.chinaz.com/dnsce/
